Makefile

```
@@ -8,7 +8,7 @@ flake8 = flake8 tests src/pytest_inmanta_yang
 .PHONY: install
 install:
 	pip install -U setuptools pip wheel
-	pip install -U --pre -r requirements.dev.txt
+	PIP_INDEX_URL=https://packages.inmanta.com/public/quickstart/python/simple/ pip install -U --pre -r requirements.dev.txt -r requirements.txt
 	pip install -e .
```

```suggestion
	PIP_INDEX_URL=https://packages.inmanta.com/public/quickstart/python/simple/ pip install -U --pre -r requirements.dev.txt -c requirements.txt -e .
```
For v2 modules the `requirements.txt` is generally a lock file.

---

Oh, right, my bad.